database_table.sql

```diff
@@ -0,0 +1,112 @@
+CREATE TABLE organizations (
+  id SERIAL PRIMARY KEY,
```

Comment:
Look up on serial and its limitations

Reply:
I read about it but didn;t get why should i change it ?